.changeset/pink-rivers-knock.md

```diff
@@ -0,0 +1,5 @@
+---
+"astro": minor
+---
+
+Astro will now notify you on start and in the dev toolbar when a new version of Astro is available. This behaviour can be configured using the new `checkUpdates` setting in the Astro config.
```

Comment:
Just checking up on this, since I don't see a config reference update to docs in this PR. Also, the blog post mentions running `astro preferences disable checkUpdates` to disable, so we should probably have a docs PR to the CLI reference page here: https://docs.astro.build/en/reference/cli-reference/#astro-preferences

Reply:
Updated the changeset with the latest info! I'll send a PR to that page.